### DISCUSSION TITLE
Appplications menu hangs in 1.5

### DISCUSSION THREAD

**Discussion by dem1980** [Rparadise-Team/Koriki#43] [General]
Hello,

i upgraded Koriki on my Miyoo Mini Plus from 1.4.2 to 1.5 and the menu Applications hangs when i open it.
Others menus are working fine (consoles, overlays, native games,...).
I have to reboot the device to use it again.
I tried to delete files from "\App\" and from "\.simplemenu\apps\" and put files from "Koriki_1.5_full.zip" but it looks the same.
It also tried to change the current them but it doesn't solve the problem.
Do you have an idea ? Or there is a way to activate some log file ?

Thank you

**Comment by Ninoh-FOX** [comment]
Erase last_state.sav from .simplemenu

**Reply by dem1980** [reply]
Well done ! It works
